projects/pastanaga-angular/src/lib/styles/_avatar.scss

```diff
@@ -15,50 +15,68 @@
 }
 
 .pa-avatar {
+    position: relative;
+    display: flex;
+    width: $avatar-small-size;
+    height: $avatar-small-size;;
```

Comment:
following the naming convention for token, I guess this should be `$size-avatar-small`